pkgs/native_assets_builder/pubspec.yaml

```diff
@@ -11,12 +11,14 @@ environment:
 
 dependencies:
   collection: ^1.18.0
+  crypto: ^3.0.6
   graphs: ^2.3.1
   logging: ^1.2.0
   # native_assets_cli: ^0.9.0
   native_assets_cli:
     path: ../native_assets_cli/
   package_config: ^2.1.0
+  xxh3: ^1.1.0
```

Comment:
Does this extra dependency on a 3rd party really provide us much value. If not, I'd consider starting without this. Any hashing algorithm can be replaced later on (if proven to be beneficial) without breaking anything.

Reply:
Oops, forgot to remove from pubspec. Done.